website/src/app/quickstart/page.tsx

```
@@ -0,0 +1,787 @@
+                        <a className="link-inline" href="https://github.com/mdhaarishussain/chaos-kitten/discussions">
+                            Discussions
+                        </a>.
+                    </p>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    {/* Go back button uses browser history */}
+    <script>
+        // you can customize behavior here if needed
```

Biome (2.4.6)

[error] Wrap comments inside children within braces.

(lint/suspicious/noCommentText)